hungphamviet/bakedfish-automation
# BF-003: Implement automation script to search for Microsoft site

## Ticket
https://github.com/hungphamviet/bakedfish-automation/issues/4

## Overview
Use BDD approach to implement the following test case:

GIVEN I am in Google Search page
WHEN I attempt to search for "Microsoft"
THEN I should see the site "Microsoft" in the first result page

## Installation Guide
Install Java 8, Geckodriver and Chromedriver (see more from previous PR: https://github.com/hungphamviet/bakedfish-automation/pull/3)

## Verification Steps
1. `git checkout ###`
2. Execute the following commands with the corresponding Gecko and Chrome driver path in your system: "mvn test -Dtest.browser=CHROME -Dwebdriver.chrome.driver=/usr/bin/chromedriver" to verify in Chrome or "mvn test -Dtest.browser=FIREFOX -Dwebdriver.gecko.driver=/usr/bin/geckodriver" to verify in Firefox
4. You should see the expected browser opens Google, Search for "Microsoft", verify Microsoft home page is in the first result,then close. And you should also see "BUILD SUCCESS" in the output of that mvn command

## Code Review Checklist

### Docs
- [ ] Are changes required for docs?
  - [ ] `readme.md` was updated?
  - [ ] Additional documentation required for `docs/`?
